src/components/Steps/Timeline.tsx

```diff
@@ -21,17 +21,40 @@ export default function Timeline({
 }: ActiveFormProps) {
   const [shouldSubmit, setShouldSubmit] = useState(false);
 
-  const { register, handleSubmit, control, reset } = useForm<QuizCreatorForm>({
+  const { register, handleSubmit, control, reset,watch } = useForm<QuizCreatorForm>({
     defaultValues: { ...data.timeline },
   });
+  const [message,setMessage] = useState("")
 
   useEffect(() => {
     shouldSubmit && OnSubmitSession!();
 
     reset();
   }, [shouldSubmit, data]);
+  function checkDateTime(startDate:Date, endDate:Date, startTime:Date, endTime:Date) {
```

Comment:
there are utility functions with respect to datetimes elsewhere. can you make use of them instead of writing code in Timeleine.tsx